### PR TITLE
Fixing struct syntax error issue #12

### PR DESCRIPTION
**Pull request by drudnev**
Fix for the struct issue

### CODE REVIEW

File: manifests/data/defaults.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+curator::package_name: 'elasticsearch-curator'
```
Comment (mvisonneau) [inline manifests/data/defaults.yaml]: is this file mandatory ? 🤔

**Comment by mvisonneau** [comment]
small comment, the rest LGTM 👍 Thanks a lot for the nice refacto! 💚 :)

File: manifests/data/defaults.yaml (hunk 1 above)
Comment (drudnev) [reply manifests/data/defaults.yaml]: Apologies this is accidental , this should of been in the root of the module. I was looking at prepping the module for puppet 5 with native hiera support for default variables, however it appears I placed defaults.yaml in the wrong path.  https://puppet.com/docs/puppet/5.0/hiera_migrate_modules.html

Comment (mvisonneau) [reply manifests/data/defaults.yaml]: ah brilliant thanks 👍looks better with thie `hiera.yaml` :)

Comment (drudnev) [reply manifests/data/defaults.yaml]: I moved to these to separate branch on for now.

**Pull request merged**